tests/manage/configuration/test_options.py

```diff
@@ -38,10 +38,10 @@ def test_get_option(self):
     def test_parse_option(self):
         """Test `parse_option` function."""
 
-        with pytest.raises(ConfigValidationError):
-            parse_option('logging.aiida_loglevel', 1)
+        with pytest.raises(ConfigurationError):
+            parse_option('logging.aiida_loglevel', {})
```

Comment:
I think this is a leftover from earlier stages of the code where this was necessary, but really because I didn't port the code correctly. I reverted the change and test still passes